package.json

```
@@ -6,7 +6,7 @@
     "dev": "NODE_ENV=development nodemon ./bin/www",
     "lint": "./node_modules/.bin/eslint **/*.js",
     "test": "NODE_ENV=test node ./node_modules/mocha/bin/mocha ./test/**/*.js --exit",
-    "migrate": "node ./bin/migrate",
+    "migrate": "node ./bin/migrate $BUILD",
```

Would this require someone running the command locally to manually set the value of `$BUILD` first? If so, can we change this to take in the argument when invoking the command instead?

---

Originally, `process.argv[2] == ""` in `bin/migrate` was supposed to take care of this, but a lack of argument turns out to be typeof undefined. Updated the migrate script to account for this as well as invalid params. Now we can run:
- `yarn run migrate`
- `yarn run migrate all`
- `yarn run migrate acs`
- `yarn run migrate decennial`

and `yarn run migrate fake` or all other params should throw an error. 

As a note, the selection and geoids always runs with a valid param.